web/src/components/FilingsCalendar.tsx

```diff
@@ -138,6 +140,23 @@ export const FilingsCalendar = (props: Props): ReactElement => {
     </div>
   );
 
+  if (props.isRoadmap) {
+    return taxFilings.length > 0 && userData?.profileData.dateOfFormation ? (
```

Comment:
why do we need the new `isRoadmap` prop?

Looking further, it seems the difference is the styling surrounding the list view? It makes me wonder if this `renderList` function should be its own component. It feels weird to have a `FilingsCalendar` component that renders two completely different things based on the roadmap prop. It's like they don't share anything except a concept of being "related to the calendar" and calling the renderList function